packages/block-library/src/template-part/edit/index.js

```diff
@@ -155,23 +162,12 @@ export default function TemplatePartEdit( {
 		[ templatePartId, attributes.area, clientId ]
 	);
 
-	const { templateParts } = useAlternativeTemplateParts(
-		area,
-		templatePartId
-	);
-	const blockPatterns = useAlternativeBlockPatterns( area, clientId );
-	const hasReplacements = !! templateParts.length || !! blockPatterns.length;
```

Comment:
I removed the `templateParts` check because it doesn't seem relevant? These templates are not included in the list.